data/en/census_household.json

```diff
@@ -3681,15 +3566,7 @@
                     "description": "",
                     "questions": [{
                         "title": "How do you usually travel to work?",
-                        "guidance": {
-                            "content": [{
-                                    "description": "Select the option for the longest part, in distance, of your journey"
-                                },
```

Comment:
should there be a line break between these two lines, i dont think there should be as they are related

Reply:
@charlotte-hirst This isn't in keeping with our patterns. We're essentially asking for guidance without the guidance box. We shouldn't be using breakpoints in the description so I think we need to discuss what is needed here.